cli/dts/lib.deno.unstable.d.ts

```diff
@@ -1214,4 +1214,46 @@ declare namespace Deno {
    * The pid of the current process's parent.
    */
   export const ppid: number;
+
+  /** **UNSTABLE**: New API, yet to be vetted.
+   * A custom HttpClient for use with `fetch`.
+   * 
+   * ```ts
+   * const client = new Deno.createHttpClient({ caFile: "./ca.pem" });
+   * const req = await fetch("https://myserver.com", { client });
+   * ```
+   */
+  export class HttpClient {
+    rid: number;
+    constructor(rid: number);
```

Comment:
The constructor can be excluded from the public interface: https://github.com/denoland/deno/blob/55ea9c7e856cae5471fea309d9dc40444581c3ae/cli/dts/lib.deno.ns.d.ts#L1850-L1851